helm-charts/seldon-core-operator/values.yaml

```diff
@@ -68,6 +68,8 @@ executor:
     defaultEndpoint: 'http://default-broker'
     workQueueSize: 10000
     writeTimeoutMs: 2000
+  # Whether to run full protocol API based health checks on models in deployment graph. False will just do TCP connects
```

Comment:
Perhaps it may be better to set to true by defualt with the updating pointing out to set it to false if required (ie they actually extend from the classes) - this may end up breaking some tests if they don't rebuild (although most of the test models should be rebuilt on the e2e test run). Or did you find that indeed quite a few tests were failing?